frontend/src/components/pages/AdminDashboard/components/CheckInInfoCard.tsx

```diff
@@ -63,7 +65,7 @@ const CheckInInfoCard = ({
         isAdmin: false,
       },
     );
```

Comment:
Can we check for response and show the appropriate toast here. If it fails we dont want to set false as the currentCheckin